js/views/userPageVw.js

```diff
@@ -457,14 +464,19 @@ module.exports = Backbone.View.extend({
       arrayItem.avatar_hash = self.model.get('page').profile.avatar_hash;
       arrayItem.handle = self.model.get('page').profile.handle;
       arrayItem.userID = self.pageID;
-
+      if (arrayItem.category != "" && $("#categories option[value='" + arrayItem.category + "']").val() === undefined){
```

Comment:
I think "$("#categories option[value='" + arrayItem.category + "']").val() === undefined" should be
"self.$el.find("#categories option[value='" + arrayItem.category + "']").length == 0"

Otherwise it could match an option that exists, but has no value assigned to it.